src/Application/AnalyzeCommand.php

```diff
@@ -29,7 +31,7 @@
 use TwigStan\Twig\TwigFileNormalizer;
 
 #[AsCommand(name: 'analyze', aliases: ['analyse'])]
-final class AnalyzeCommand extends Command
+class AnalyzeCommand extends Command
```

Comment:
Why this change

Reply:
Sorry about that. It was a try but now I can restore as before